packages/react-components/react-popover/src/components/Popover/usePopover.ts

```diff
@@ -121,7 +121,10 @@ export const usePopover_unstable = (props: PopoverProps): PopoverState => {
 
   React.useEffect(() => {
     if (open && positioningRefs.contentRef.current) {
-      const firstFocusable = findFirstFocusable(positioningRefs.contentRef.current);
+      const firstFocusable =
+        positioningRefs.contentRef.current.tabIndex !== -1
```

Comment:
This should be `positioningRefs.contentRef.current.tabIndex === undefined`, I believe. If the `tabindex="-1"`, we'd still want to focus it by default, since it's focusable and `-1` would actually be the correct value if someone wanted to focus the container. It's only if the `contentRef` is fully not focusable that we wouldn't want to focus it.